vspace/src/main/webapp/WEB-INF/views/staff/user/list.html

```diff
@@ -1,47 +1,47 @@
-		        
-		        </div>
-		    </div>
-	</div>
+<html layout:decorate="~{layouts/main_staff}">
+<head>
+</head>
+
+<body>
+	<div layout:fragment="content" class="main-content">
+		<h2>Users</h2>
+        <div  th:each="user: ${users}" class="tab" style="display: flex; justify-content: space-between">
+		    	<h5 th:if="${#lists.contains(user.roles, 'STAFF')}" class="badge badge-primary">Staff</h5>
+		        [[${user.username}]] ([[${user.firstName}]] [[${user.lastName}]])
+		        <div>
+		        	<span th:if="${!user.enabled}" class="badge badge-light">Account Deactivated</span>
+			        <span th:if="${user.admin}" class="badge badge-info" style="background: var(--primary)">Admin User</span>
+			        <span th:if="${user.enabled}" class="badge badge-success">Active</span>
+		        </div>
+		        
+		        <div>
+		        
+	            <form th:if="${!user.enabled}" th:action="@{'/staff/user/'+${user.username}+'/approve'}"  th:object="${approveUrl}" method="POST">
+	            	<button style="padding:0" class="btn btn-link"><span><i class="fas fa-user-check"></i></span> Activate</button>
+	            </form>
+		        <div th:if="${user.enabled}" style="display: flex">
+                    <form th:if="${!user.admin}" th:action="@{'/staff/user/'+${user.username}+'/admin'}" th:object="${postUrl}" method="POST">
+                        <button style="padding:0" class="btn btn-link"><span><i class="fas fa-key"></i></span> Make Admin</button>
+                    </form>
+                    <form th:if="${user.admin}" th:action="@{'/staff/user/'+${user.username}+'/admin/remove'}" th:obejct="${removeUrl}" method="POST">
+                    <button style="padding:0" class="btn btn-link"><span><i class="fas fa-user"></i></span> Remove Admin Role</button>
+                    </form>
+                    
+                    <form th:if="${!#lists.contains(user.roles, 'STAFF')}" th:action="@{'/staff/user/'+${user.username}+'/role/add?roles=ROLE_STAFF'}" th:object="${url}" method="POST">
+                        <button style="padding:0" class="btn btn-link"><span><i class="fas fa-user"></i></span> Make Staff</button>
+                    </form>
+                    
+                    <form th:if="${#lists.contains(user.roles, 'STAFF')}" th:action="@{'/staff/user/'+${user.username}+'/role/remove?roles=ROLE_STAFF'}" th:object="${url}" method="POST">
+                        <button style="padding:0" class="btn btn-link"><span><i class="fas fa-user"></i></span> Remove Staff Role</button>
+                    </form> 
+                
+                    <form th:action="@{'/staff/user/'+${user.username}+'/disable'}" th:object="${disableUrl}" method="POST">
+                        <button style="padding:0" class="btn btn-link"><span><i class="fas fa-user-slash"></i></span> Deactivate User</button>
+                    </form>
+                </div>
+		        
+		        </div>
+		    </div>
+	</div>
```

Comment:
same here? why does it show up as completely changed? what has actually changed?

Reply:
and this one doesn't have all lines changed (the last two stay unchanged), so it can't be line endings.

Reply:
?